test/extended/util/image/zz_generated.txt

```diff
@@ -1,4 +1,5 @@
 # This file is generated by hack/update-generated-bindata.sh
+docker.io/centos/httpd-24-centos8:latest quay.io/openshift/community-e2e-images:e2e-docker-io-centos-httpd-24-centos8-latest-00vUTwFFxuIiiWjJ
```

Comment:
you need to rerun "make update" or whatever to regenerate this file. (well, or just revert the change to it)